slasher/usage.go

```diff
@@ -5,7 +5,6 @@ import (
 	"io"
 	"sort"
 
-	"github.com/prysmaticlabs/prysm/shared/featureconfig"
```

Comment:
lol what happened here

Reply:
My bad @0xKiwi apologize for the bad commit

Reply:
No worries @rauljordan ! Was funny haha